pkg/operator/workload/workload_openshiftapiserver_v311_00_sync.go

```diff
@@ -475,3 +413,9 @@ func proxyMapToEnvVars(proxyConfig map[string]string) []corev1.EnvVar {
 	sort.Slice(envVars, func(i, j int) bool { return envVars[i].Name < envVars[j].Name })
 	return envVars
 }
+
+func logError(_ *operatorv1.OperatorStatus, _ bool, err error) {
+	if err != nil {
+		klog.Info(err)
```

Comment:
I need to check how we usually log errors (log level) and consider prefixing it with `Failed to update the operator status due to %v`